.cirrus.yml

```diff
@@ -17,7 +17,7 @@ alpine-3_task:
     cpu: 1
   fix-pyproject-metadata_script:
     # see https://github.com/FFY00/python-pyproject-metadata/pull/44
-    - python3 -m pip install "pyproject-metadata < 0.7.0"
+    - python3 -m pip install "pyproject-metadata != 0.7.0"
```

Comment:
Just drop the work-around. There is no need to keep it. The latest release will be installed anyhow.

Reply:
I mean to say that you should just drop the `fix-pyproject-metadata_script` block.